charts/resources/instance/templates/instance.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: servicecatalog.k8s.io/v1alpha1
+kind: Instance
+metadata:
+  name: {{ .Values.name }}
+spec:
+  serviceClassName: {{ .Values.serviceClassName }}
+  planName: {{ .Values.planName }}
```

Comment:
I have to echo @vaikas-google's questions. Even though you have addressed them already, @arschles, I don't feel I understand.

Let me say that I _do_ see the value and the power behind some future demo wherein, for instance, an instance and a binding could _both_ be included in a single chart and service catalog will sort everything out and the whole system will be eventually consistent. (To make this even more concrete, imagine augmenting a chart like the existing Drupal or Wordpress charts to use instance and binding resources to express a dependency on an rdbms without much regard for _how_ it's provisioned.)

But if these new charts, as PR'ed here, just contain a single resource each, aren't they not doing anything more than what the example manifests already do?